test/e2e/integreatly_test.go

```diff
@@ -169,65 +166,6 @@ func waitForProductDeployment(t *testing.T, f *framework.Framework, ctx *framewo
 	return nil
 }
 
-func execToPod(command string, podname string, namespace string, container string, f *framework.Framework) (string, error) {
```

Comment:
just out of curiosity why is this being removed? is this being replaced by this one https://github.com/integr8ly/integreatly-operator/blob/master/test/common/shared_functions.go#L38

Reply:
Yup, the same function got moved there for the tests in `test/common` package and this function wasn't anymore in `test/e2e/integreatly_test.go` so just thought i'll remove while I was there